Vic2ToHoI4/Source/V2World/Diplomacy/Relations.h

```diff
@@ -0,0 +1,44 @@
+#ifndef VIC2_RELATIONS_H
+#define VIC2_RELATIONS_H
+
+
+
+#include "Date.h"
+#include <optional>
+
+
+
+namespace Vic2
+{
+
+class Relations
+{
+  public:
+	class Factory;
+	class Builder;
+
+	[[nodiscard]] int getRelations() const { return value; }
+	[[nodiscard]] int getLevel() const { return level; }
+
+	[[nodiscard]] bool hasMilitaryAccess() const { return militaryAccess; }
+	[[nodiscard]] const auto& getDiplomatLastSent() const { return lastSentDiplomat; }
+	[[nodiscard]] const auto& getLastWar() const { return lastWar; }
+	[[nodiscard]] const auto& getTruceUntil() const { return truceUntil; }
+	[[nodiscard]] int getInfluenceValue() const { return influenceValue; }
+
+  private:
+	int value = 0;
+	int level = 2;
```

Comment:
Why 2? A comment would be nice.

Reply:
Replaced with an enum so such things are more clear.